automata/fa/dfa.py

```diff
@@ -148,6 +150,23 @@ def __invert__(self):
         """Return the complement of this DFA and another DFA."""
         return self.complement()
 
+    def __contains__(self, other):
+        return self.accepts_input(other)
+
+    def __iter__(self):
+        i = self.minimum_word_length()
+        limit = self.maximum_word_length()
+        while i <= limit:
```

Comment:
@Tagl I would much prefer a `for` loop here to eliminate the manual counting:

```python
def __iter__(self):
    min_len = self.minimum_word_length()
    max_len = self.maximum_word_length()
    for i in range(min_len, max_len + 1):
        yield from self.words_of_length(i)
```

Also, props to you for using `yield from`

Reply:
The reason for the while loop is that `limit` can be `float('inf')`

Reply:
@Tagl I see. `range()` can't receive `float` values, so I understand the need to use a `while` now.